docs/content/concepts/sharding.md

```diff
@@ -0,0 +1,204 @@
+---
+description: >
+    Horizontal Scaling of kcp through sharding
+---
+
+# Shards
+
+Every kcp shard is hosting a set of logical clusters. A logical cluster is
+identified by a globally unique identifier called a *cluster name*. A shard
+serves the logical clusters under `/clusters/<cluster-name>`.
+
+A set of known shards comprises a kcp installation.
+
+# Root Logical Cluster
+
+Every kcp installation has one logical cluster called `root`. The `root` logical
+cluster holds administrational objects. Among them are shard objects.
+
+# Shard Objects
+
+The set of shards in a kcp installation is defined by `Shard` objects in
+`core.kcp.io/v1alpha1`.
+
+A shard object specifies the network addresses, one for external access (usually 
+some worldwide load balancer) and one for direct access (shard to shard).
+
+# Logical Clusters and Workspace Paths
+
+Logical clusters are defined through the existence of a `LogicalCluster` object
```

Comment:
Can shard host more than 1 logical cluster? Or logical cluster and shard are 1:1 mapping?